pkg/apiserver/api/v1/instance.proto

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package kubecarrier.api.v1;
+option go_package = "v1";
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+import "meta.proto";
+
+message Instance {
+  // Metadata - common metadata
+  ObjectMeta metadata = 1;
+  // Offering name, i.e. couchdb.eu-west-1.team-a
+  string offering = 2;
+  // Spec - instance spec
+  RawObject spec = 3;
+  // Status - instance status
+  RawObject status = 4;
+}
+
+message InstanceList {
+  ListMeta metadata = 1;
+  repeated Instance items = 2;
+}
+
+message InstanceGetRequest {
```

Comment:
Since PR #426 is merged, can you please reuse/extend the Get/ListRequest in Service API?

Reply:
but Service API has additional params, that no present in `Get/ListRequest`

Reply:
I think you can extend Get/ListRequest with new params. I also use Get/ListRequest in the Account PR.

Reply:
yes, because account service doesn't have any additional fields

Reply:
OK, so what is the reason to have a general List/GetRequest? I don't see any reason that we can not extend it to include params and use it everywhere